samples/features/azure-arc-data-controller/deployment/kubeadm/ubuntu-single-node-vm/README.md

```diff
@@ -0,0 +1,47 @@
+
+# Deploy a Azure Arc Data Controller on single node Kubernetes cluster (kubeadm)
+
+Using this sample bash script, you will deploy a single node Kubernetes cluster using  kubeadm and a Azure Arc Data Controller on top of it. The script must be run from the VM you are planning to use for your kubeadm deployment.
+
+## Pre-requisites
+
+1. A vanilla Ubuntu 16.04 or 18.04 virtual or physical machine. All dependencies will be setup by the script. Using Azure Linux VMs is not yet supported.
+1. Machine should have at least 8 CPUs, 64GB RAM and 100GB disk space. After installing the images you will be left with 50GB for data/logs across all components.
+1. Update existing packages using commands below to ensure that the OS image is up to date
+
+``` bash
+sudo apt update&&apt upgrade -y
+sudo systemctl reboot
+```
+
+## Recommended Virtual Machine settings
+
+1. Use static memory configuration for the virtual machine. For example, in hyper-v installations do not use dynamic memory allocation but instead allocate the recommended 64 GB or higher.
+
+1. Use checkpoint or snapshot capability in your hyper visor so that you can rollback the virtual machine to a clean state.
+
+## Instructions to deploy Azure Arc Data Controller
+
+1. Download the script on the VM you are planning to use for the deployment
+
+``` bash
+curl --output setup-controller.sh https://raw.githubusercontent.com/ananto-msft/sql-server-samples/master/samples/features/azure-arc-data-controller/deployment/kubeadm/ubuntu-single-node-vm/setup-controller.sh
```

Comment:
You will have to update the raw git link based on the suggested folder structure.

Reply:
Done.